runtime/rasdump/javadump.cpp

```diff
@@ -314,6 +314,7 @@ private :
 	IDATA       getOwnedObjectMonitors       (J9VMThread* vmThread, J9ObjectMonitorInfo* monitorInfos);
 	void 		writeJavaLangThreadInfo		 (J9VMThread* vmThread);
 	void		writeCPUinfo				 (void);
+ 	void 		writeCgroupMetrics				 (void);
```

Comment:
While this code is already a mess of mismatched formatting, please use this form for new lines here:
```
void 		writeCgroupMetrics(void);
```

Normally I would object to the tabs between the return type and the method name, but it matches the convention in this code

Reply:
@DanHeidinga Thanks for the review. I have formatted as per your comments.